rerun_cpp/src/rerun/datatypes/mat4x4_ext.cpp

```diff
@@ -37,6 +37,29 @@ namespace rerun {
                       columns[3].w(),
                   } {}
 
+            /// Construct a new 4x4 matrix from a pointer to 16 floats (in row major order).
+            ///
+            /// Attention: The pointer must point to at least least 16 floats long.
```

Comment:
Ignoring the the duplicated word, this is superfluous. You already state `pointer to 16 floats`

```suggestion
```